keras_cv/metrics/coco/recall.py

```diff
@@ -151,6 +156,8 @@ def update_state(self, y_true, y_pred, sample_weight=None):
             dtype=self.compute_dtype,
         )
 
+        y_pred = utils.sort_bounding_boxes(y_pred, axis=bounding_box.XYXY.CONFIDENCE)
```

Comment:
Is this going to fix also https://github.com/keras-team/keras-cv/issues/328 ?

Reply:
yep

Reply:
Do we have gained only -0.01  with al these fixes?

Reply:
yep

Reply:
in extreme cases though it saves us potentially a result that should be 1.0 showing as 0.0